modules/apps/callbacks/go.mod

```diff
@@ -58,6 +57,7 @@ require (
 	cosmossdk.io/x/evidence v0.0.0-00010101000000-000000000000 // indirect
 	cosmossdk.io/x/group v0.0.0-00010101000000-000000000000 // indirect
 	cosmossdk.io/x/nft v0.0.0-00010101000000-000000000000 // indirect
+	cosmossdk.io/x/params v0.0.0-00010101000000-000000000000 // indirect
```

Comment:
Should there be anything left to keep this around?

Reply:
nope - can remove